src/webhint-theme/helper/utils.js

```diff
@@ -77,12 +77,6 @@ module.exports = {
             return page.section === section;
         });
     },
-    getSignalIssueQuery: (root, title, directory) => {
-        const issueTitle = `[docs] Issue with '${title}'`;
-        const issueContent = url.resolve(root, directory);
```

Comment:
FYI, we're failing CI because linting is now complaining that `url` is no longer used (as this was the only usage). If you remove the definition from the top of the file that should take care of it.

```js
const url = require('url');
```